### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by pbsds** [mastodon/mastodon-android#334]
The mastodon app crashes after i open it, every time. I tried wiping cache and data, then logging back in again. The crashing behavior started again right after i logged back in again, indicating that some state or package from the server is likely the cause.

Output from `adb logcat | grep -i mastodon`: https://gist.github.com/pbsds/e5acd605799680ba2e24e48b7c8e33aa

My homeserver is snabelen.no, which at time of writing runs mastodon v3.5.3

**Comment by insertjokehere** [comment]
I'm also seeing the Mastodon app crash every time I open it. Not sure if this is the same issue, trace doesn't look the same:

```
11-14 10:44:24.901  1883 14323 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=org.joinmastodon.android/.MainActivity bnds=[436,345][643,675]} from uid 10154
11-14 10:44:24.929  1883  2096 I ActivityManager: Start proc 11782:org.joinmastodon.android/u0a48 for next-top-activity {org.joinmastodon.android/org.joinmastodon.android.MainActivity}
11-14 10:44:24.932  1883 14325 D CoreBackPreview: Window{716d42b u0 Splash Screen org.joinmastodon.android}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@110fa21, mPriority=0}
11-14 10:44:24.983 11782 11782 W ziparchive: Unable to open '/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.en.dm': No such file or directory
11-14 10:44:24.983 11782 11782 W ziparchive: Unable to open '/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.en.dm': No such file or directory
11-14 10:44:24.984 11782 11782 W ziparchive: Unable to open '/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.fr.dm': No such file or directory
11-14 10:44:24.984 11782 11782 W ziparchive: Unable to open '/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.fr.dm': No such file or directory
11-14 10:44:24.984 11782 11782 W ziparchive: Unable to open '/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.xxhdpi.dm': No such file or directory
11-14 10:44:24.984 11782 11782 W ziparchive: Unable to open '/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.xxhdpi.dm': No such file or directory
11-14 10:44:24.985 11782 11782 D nativeloader: Configuring classloader-namespace for other apk /data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/base.apk:/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.en.apk:/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.fr.apk:/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/split_config.xxhdpi.apk. target_sdk_version=31, uses_libraries=, library_path=/data/app/~~MfwJsKjMVGk_TPQJLaBZmg==/org.joinmastodon.android--TLbCYcbFel2DOjcfWExkA==/lib/arm64, permitted_path=/data:/mnt/expand:/data/user/0/org.joinmastodon.android
11-14 10:44:24.993 11782 11782 V GraphicsEnvironment: ANGLE Developer option for 'org.joinmastodon.android' set to: 'default'
11-14 10:44:24.993 11782 11782 V GraphicsEnvironment: ANGLE GameManagerService for org.joinmastodon.android: false
11-14 10:44:25.204  1883 14326 D CoreBackPreview: Window{ea180f7 u0 org.joinmastodon.android/org.joinmastodon.android.MainActivity}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@5fc91cd, mPriority=0}
11-14 10:44:25.288  1883  2079 I ActivityTaskManager: Displayed org.joinmastodon.android/.MainActivity: +380ms
11-14 10:44:25.324 11782 11800 E AndroidRuntime: Process: org.joinmastodon.android, PID: 11782
11-14 10:44:25.324 11782 11800 E AndroidRuntime:        at org.joinmastodon.android.api.PushSubscriptionManager$a.lambda$onSuccess$0(:1)
11-14 10:44:25.324 11782 11800 E AndroidRuntime:        at org.joinmastodon.android.api.PushSubscriptionManager$a.a(Unknown Source:0)
11-14 10:44:25.324 11782 11800 E AndroidRuntime:        at org.joinmastodon.android.api.k0.run(Unknown Source:4)
11-14 10:44:25.326  1883 14326 W ActivityTaskManager:   Force finishing activity org.joinmastodon.android/.MainActivity
11-14 10:44:25.373  1883  3143 I WindowManager: WIN DEATH: Window{ea180f7 u0 org.joinmastodon.android/org.joinmastodon.android.MainActivity}
11-14 10:44:25.374  1883  3143 W InputManager-JNI: Input channel object 'ea180f7 org.joinmastodon.android/org.joinmastodon.android.MainActivity (client)' was disposed without first being removed with the input manager!
11-14 10:44:25.375  1883 14325 I ActivityManager: Process org.joinmastodon.android (pid 11782) has died: fg  TOP
11-14 10:44:25.391  1883 14325 W WindowManager: Failed to deliver inset state change to w=Window{ea180f7 u0 org.joinmastodon.android/org.joinmastodon.android.MainActivity EXITING}
11-14 10:44:25.488  1883 14325 W WindowManager: Exception thrown during dispatchAppVisibility Window{ea180f7 u0 org.joinmastodon.android/org.joinmastodon.android.MainActivity EXITING}
11-14 10:44:25.494  1883 14325 W InputManager-JNI: Input channel object '716d42b Splash Screen org.joinmastodon.android (client)' was disposed without first being removed with the input manager!
11-14 10:44:25.828  1883  2084 W ActivityTaskManager: Activity top resumed state loss timeout for ActivityRecord{3b7e60f u0 org.joinmastodon.android/.MainActivity} t-1 f}}
```

Mastodon server v3.5.3, Mastodon Android v1.1.3, Google Pixel 4a running Android 13. I'm using OAuth to auth with the server, crashes as soon as I press the "Authorize" button and won't start until I clear data from the app

**Comment by pbsds** [comment]
It's become a bit random when it happens, but i believe a toot might be to blame, as today it only crashed after i scrolled a few posts, reproducibly so.

**Comment by grishka** [comment]
Might be related to #275. I probably need to publish a new beta build.

**Comment by insertjokehere** [comment]
I'm still seeing this issue - Mastodon Android v1.2.3, Mastodon server v4.1.2

**Comment by grishka** [comment]
I still don't have a slightest idea about what this is about.

**Comment by Coehill** [comment]
My mastodon app started crashing every time I start it up on version 2.5.4.  I have tried force stopping, clearing cache, and restarting my Android device.

**Comment by grishka** [comment]
You should update.

**Comment by Coehill** [comment]
> You should update.

Unfortunately, I have the latest version available in F-Droid so I cannot uodate. I think if the build was reproducible, then I'd be able to update from source in GitHub or even Google Play.

**Comment by quintus3236** [comment]
2.7.0 Same issue here. (IzzyOnDroid) I click the icon then just says "Mastodon is not responding"

LineageOS 20, Android 13

**Comment by grishka** [comment]
Did someone again pick up one of the bad builds that were available for like an hour at night?..

**Comment by grishka** [comment]
Fixed in 2.7.1, please install this build in the meantime: https://github.com/mastodon/mastodon-android/actions/runs/11170110424/artifacts/2013267341
(IzzyOnDroid will host this exact apk eventually)
